tree-sitter-stack-graphs/src/cli/index.rs

```diff
@@ -428,10 +428,19 @@ impl<'a> Indexer<'a> {
         cancellation_flag: &dyn CancellationFlag,
     ) -> std::result::Result<(), BuildErrorWithSource<'b>> {
         let relative_source_path = source_path.strip_prefix(source_root).unwrap();
+        // here the file should also have stripped the source_root from its path
```

Comment:
I don't think we want to use the relative path for the `file`. This would very easily lead to conflicts, for example if you index multiple source directories with the CLI. But if we set the global variables explicitly, it won't be an issue that the scope graph records the full path.

Reply:
yep 100% agreed, this is an old comment I forgot to remove